data/korath/korath.txt

```diff
@@ -242,6 +256,32 @@ fleet "Korath Ember Waste Raid"
 		"Rano'erek (Shunt)"
 		"'olofez" 4
 
+fleet "Kor Alaji Raid"
+	government "Kor Alaji"
+	names "korath"
+	fighters "korath"
+	cargo 1
+	commodities "Food" "Clothing" "Metal" "Plastic" "Medical" "Heavy Metals"
+	outfitters "Korath Exiles Remnant Donations"
+	personality
+		disables plunders opportunistic harvests
+	variant 12
+		"Palavret (Ember)"
+		"'olofez" 2
+	variant 3
+		"Palavret (Ember Blaze)"
+		"'olofez" 2
+	variant 6
+		"Palavret (Ember Shunt)"
+		"'olofez" 2
+	variant 1
+		"Rano'erek"
+		"'olofez" 4
+	variant 3
+		"Rano'erek (Shunt)"
+		"'olofez" 4
```

Comment:
Why does this need to be a new fleet? Why not just change the government of the fleet you're cloning and not need to update every reference to the fleet?

Reply:
It is for random period fleets, who don't get to specify their government.